extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -286,11 +333,32 @@ export class Bind {
     // Helper function for scanning the tree walker's next node.
     // Returns true if the walker has no more nodes.
     const scanNextNode_ = () => {
-      const element = walker.nextNode();
+      const element = walker.currentNode;
       if (!element) {
         return true;
       }
       const tagName = element.tagName;
+      if (DYNAMIC_TAGS.includes(tagName)) {
```

Comment:
Explanation: we were always immediately calling #nextNode(), which caused us to ignore the root node. This worked fine when we were always scanning with the body element as the root.